src/lib/runtime_config/runtime_config.ml

```diff
@@ -1,9 +1,22 @@
 open Core_kernel
 
 module Fork_config = struct
+  (* Note that previous_length might be smaller than the gernesis_slot
+     or equal if a block was produced in every slot possible. *)
   type t =
-    { previous_state_hash : string; previous_length : int; genesis_slot : int }
-  [@@deriving yojson, dhall_type, bin_io_unversioned]
```

Comment:
@ghost-not-in-the-shell Whilst cherry picking I missed out the dhall_type. Does this need to be added in? Thanks.

Reply:
I think dhall_type is not implemented on develop branch, so you can safely change it to `[@@deriving yojson, bin_io_unversioned]`.

Reply:
Thanks. I'll leave it as is then (with the dhall_type omitted).